### ISSUE TITLE
Could you provide a script to generate training data, like Bert "create_pretraining_data.py"

### ISSUE THREAD

**Issue by yuye2133** [google-research/pegasus#103]

**Comment by JingqingZ** [comment]
Hi, I am sorry we don't have the plan to develop the script in python. The c++ ops to generate masks for pretraining has been implemented in https://github.com/google-research/pegasus/blob/939830367bcf411193d2b5eca2f2f90f3f9260ca/pegasus/ops/pretrain_parsing_ops.cc#L294.

**Comment by yuye2133** [comment]
But I don’t see the main function in the "pegasus/pegasus/ops/pretrain_parsing_ops.cc"， how to generate pretrain file.
You are missing the steps and environment for compiling c++
**_pretrain_parsing_ops.cc:26:33: fatal error: base/integral_types.h: No such file or directory_**

**Comment by leewonmin** [comment]
> 하지만 "pegasus / pegasus / ops / pretrain_parsing_ops.cc"에서 프리 트레인 파일을 생성하는 방법에 대한 주요 기능을 찾을 수 없습니다.
> C ++
> **_pretrain_parsing_ops.cc:26:33 : 치명적 오류 : base / integral_types.h : 해당 파일 또는 디렉토리가 없습니다._**

did you solve the problem? if you have solved the problem, can you tell me how?

**Comment by Arij-Aladel** [comment]
@JingqingZ 

> Hi, I am sorry we don't have the plan to develop the script in python. The c++ ops to generate masks for pretraining has been implemented in
> 
> https://github.com/google-research/pegasus/blob/939830367bcf411193d2b5eca2f2f90f3f9260ca/pegasus/ops/pretrain_parsing_ops.cc#L294
> .

what are the steps to generate pretrained data to use it in another model let us say(T5)? can you provide thw steps please?
